### ISSUE TITLE
Enum field type in model and spelling error in select 

### ISSUE THREAD

**Issue by dblanco-dev** [TiagoSilvaPereira/vemto-issues#170]
There is a spelling error when trying to modify an item in a select in the CRUD section:

![Snipaste_2021-06-17_11-54-14](https://user-images.githubusercontent.com/33554631/122481564-22bb3f80-cf8c-11eb-9aaf-c9be46437592.png)

In the CRUD section, when adding a select bound to a field of a model, it allows adding items but they cannot be modified or deleted. (You can see it in the image above).

Also I have this other error, I did the following:
1. I added a model with an integer type field
2. I added a CRUD from the previous model
3. I modified the CRUD (size of inputs, position, validations)
4. I modified the model, changed the integer to an ENUM and added options to it.
5. I generated the project and did tests
6. I closed Vemto and opened it again
7. I modified the model so that the field was again an integer
8. I modified the CRUD and it wouldn't let me remove the options until I removed the select input
9. I generated the project

It does not let me generate the project again because it tells me that it cannot find "name", I do not understand the error because the CRUD does not have any "name" field.

Also, when I generate the project the integer model field in the migration has the above enum options.

**Comment by TiagoSilvaPereira** [comment]
Hi @dblanco-dev thank you for reporting. Could you please send a screenshot showing this error:

```
It does not let me generate the project again because it tells me that it cannot find "name", I do not understand the error because the CRUD does not have any "name" field.
```

**Comment by dblanco-dev** [comment]
> Hi @dblanco-dev thank you for reporting. Could you please send a screenshot showing this error:
> 
> ```
> It does not let me generate the project again because it tells me that it cannot find "name", I do not understand the error because the CRUD does not have any "name" field.
> ```

Sorry I deleted the project and re-created it after failing to generate the code, today I tried to replicate the error but couldn't.

But I am attaching some screenshots of the other error that I mentioned:

![Snipaste_2021-06-18_09-11-18](https://user-images.githubusercontent.com/33554631/122583405-6c506c80-d016-11eb-9386-ca0572ac51e0.png)

![Snipaste_2021-06-18_09-21-04](https://user-images.githubusercontent.com/33554631/122583497-88540e00-d016-11eb-9fae-83778d73d1d3.png)

![Snipaste_2021-06-18_09-12-19](https://user-images.githubusercontent.com/33554631/122583536-96099380-d016-11eb-80fd-9ae6c8396a89.png)

![Snipaste_2021-06-18_09-15-13](https://user-images.githubusercontent.com/33554631/122583552-9c980b00-d016-11eb-9a08-151e9c6dd2a2.png)

**Comment by TiagoSilvaPereira** [comment]
Perfect @dblanco-dev I'll fix it ASAP

**Comment by TiagoSilvaPereira** [comment]
Partially fixed on 0.9.7

**Comment by TiagoSilvaPereira** [comment]
Finished on 0.9.8
